<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1367</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="firstCityLbl">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>First City</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondcityLbl">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Second City</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FirstCityCmb">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="secondCityCmb">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="MapSelectionCmb">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstCityLbl_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exploreButton">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>100</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Explore Graph</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>1081</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addGraphButton">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>170</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteGraphButton">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>240</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Graph</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1367</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
